Swiften/Elements/ClientState.h

```diff
@@ -18,14 +18,18 @@ namespace Swift {
 
         public:
             enum ClientStateType {Active, Inactive};
-            ClientState(ClientStateType state = Active) {
+            ClientState() {
+
+            }
```

Comment:
Why did you introduce this ctor and didn't just leave the other one with the default parameter?